tools/ports/libjpeg.py

```diff
@@ -17,7 +17,7 @@ def get(ports, settings, shared):
   # Archive mirrored from http://www.ijg.org/files/jpegsrc.v9c.tar.gz.
```

Comment:
Can you update this comment too?

I just uploaded jpegsrc.v9f.tar.gz to https://storage.googleapis.com/webassembly/emscripten-ports/jpegsrc.v9f.tar.gz